### DISCUSSION TITLE
Over 200 classes were generated for component styled.div

### DISCUSSION THREAD

**Discussion by ssvendsen** [glideapps/glide-data-grid#512] [Q&A] [answered]
Hi. 
I get the warning above in the console (in dev mode) when Glide.DataEditor renders a sufficient number of times. To specific I see it when resizing columns, which in my implementation causes a state change and render. Is this a known thing, and is there something I can do to address it?  I am on version 4.2.
Thanks!

**Comment by jassmith** [comment, marked as answer]
5.x series no longer uses styled-components and thus avoids this issue. The port forward should be pretty easy.

**Reply by ssvendsen** [reply]
I will try that, thanks :)
